src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -132,6 +132,7 @@
                       :showErrors="showErrors"
                       :businessLookup="inProgressBusinessLookup"
                       :BusinessLookupServices="BusinessLookupServices"
+                      :editableBusinessName="wasReplaced(currentOrgPerson)"
```

Comment:
The current org-person is given the REPLACED action [here](https://github.com/bcgov/business-edit-ui/blob/a3a1416a82d14de2c299e21f04ec18e8ca4a5a1b/src/components/common/PeopleAndRoles/PeopleAndRoles.vue#L639) if the replace event was emitted, which is only possible if the user is staff, this is a firm change filing, and this is a proprietor and an org (ie, DBA). See [canReplace()](https://github.com/bcgov/business-edit-ui/blob/a3a1416a82d14de2c299e21f04ec18e8ca4a5a1b/src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue#L480).